src/frontend/js/toc.js

```diff
@@ -460,26 +460,121 @@ function setUpNotes() {
     });
 }
 
-//  Initial setup
+// Accessibility addition to close TOC and move focus to selected item within the reader frame.
+// Uses click event because screen readers since they use simulated click events,
+// even if keyboard navigation is used.
+function tocClickHandler() {
+    'use strict';
+
+    $(TOC_CONTAINER).on('click', 'a:not([data-cfw="collapse"])', function(event) {
```

Comment:
There's another click handler for - I think - the same set of links above at line 159. Would it make sense to combine them so the code is that much more clear?